### PR TITLE
Support phone number cases like "000 111 82-2100"

### PR DESCRIPTION
**Pull request by Irrelevances**

### CODE REVIEW

File: .NET/Microsoft.Recognizers.Definitions.Common/BasePhoneNumbers.cs [hunk 1]
```diff
@@ -1,55 +1,55 @@
-//------------------------------------------------------------------------------
```
Comment (tellarin) [inline .NET/Microsoft.Recognizers.Definitions.Common/BasePhoneNumbers.cs]: Please be careful to not change all line endings in a file if not really necessary. Can you undo this?

File: .NET/Microsoft.Recognizers.Definitions.Common/BasePhoneNumbers.cs [hunk 2]
```diff
@@ -1,55 +1,55 @@
...
+//------------------------------------------------------------------------------
+namespace Microsoft.Recognizers.Definitions
+{
+	using System;
+	using System.Collections.Generic;
+
+	public static class BasePhoneNumbers
+	{
+		public const string NumberReplaceToken = @"@builtin.phonenumber";
+		public const string GeneralPhoneNumberRegex = @"(\b(((\d[\s]?){4,12}))(-?[\d\s?]{3}\d)(?!-)\b)|(\(\d{5}\)\s?\d{5,6})|\+\d{2}\(\d\)\d{10}";
```
Comment (tellarin) [inline .NET/Microsoft.Recognizers.Definitions.Common/BasePhoneNumbers.cs]: I don't think this is the right place to add support for this number format. It doesn't look like a general/common format to me.

File: Specs/Sequence/English/PhoneNumberModel.json [hunk 3]
```diff
@@ -1,4 +1,19 @@
...
+  {
+    "Input": "Tel: 000 111 82-2100.",
+    "Results": [
+      {
+        "Text": "000 111 82-2100",
+        "Start": 5,
+        "End": 19,
+        "TypeName": "phonenumber",
+        "Resolution": {
+          "score": "1",
```
Comment (tellarin) [inline Specs/Sequence/English/PhoneNumberModel.json]: You can see the side effect here. This is not a typical number, so it should not have a 1.0 score.
Perhaps SpecialPhoneNumberRegex or somewhere else is a better place to match it. As for the appropriate score, take a look at the other spec cases to see if the score given it to it makes sense compared to others.

File: .NET/Microsoft.Recognizers.Definitions.Common/BasePhoneNumbers.cs (hunk 2 above)
Comment (Irrelevances) [reply .NET/Microsoft.Recognizers.Definitions.Common/BasePhoneNumbers.cs]: If I made a common regex like "(\b(((\d[\s-]?){7,15}\d))(?!-)\b)|(\(\d{5}\)\s?\d{5,6})|\+\d{2}\(\d\)\d{10}",  "12-12-2012" was recognized as a phone number, how can I distinguish them?

**Pull request merged**